service/render-test-config.sh

```diff
@@ -5,7 +5,7 @@
 
 
 VAULT_TOKEN=${1:-$(cat $HOME/.vault-token)}
-DSDE_TOOLBOX_DOCKER_IMAGE=broadinstitute/dsde-toolbox:consul-0.20.0
+DSDE_TOOLBOX_DOCKER_IMAGE=broadinstitute/dsde-toolbox:dev
```

Comment:
@aherbst-broad did you intend to leave this as dev?

Reply:
Yes, that is the tag we use elsewhere to get the latest version of dsde-toolbox. The consul-0.20.0 tag was not working for me locally.